ADR/0025-container-images.md

```diff
@@ -0,0 +1,94 @@
+# 25. Container Image Process
+
+Date: 2023-05-26
+
+## Status
+
+Proposed
+
+## Context
+
+The purpose of this document is to establish container image management practices for AppStudio container images that are deployed in the staging and production environments.  The goal is to ensure that AppStudio is continuously maintaining secure operations that are in accordance with the ESS SEC-PATCH-REQ-2 (OS Patching) requirements.
+
+### Scope
+* The scope of this process is limited to the images found in our [quay.io/organization/redhat-appstudio](https://quay.io/organization/redhat-appstudio) repository.
+* Images from dependencies that fall outside of this AppStudio process should follow the ESS SEC-PATCH-REQ-1 and ESS SEC-PATCH-REQ-2 processes.  It is up to the component teams to ensure they are adhering to these requirements.
```

Comment:
I hesitated adding `The Source` links because they are internal.  I think there may be an ESS doc for external consumption, I'll try to confirm if I can use it

Reply:
Waiting on confirmation from the Infosec team

Reply:
Any update @kim-tsao?

Reply:
@ralphbean yes, there is a link we can share.  I can update this